draft-ietf-quic-http.md

```diff
@@ -472,145 +475,142 @@ or complete response but does not abort reading, clients SHOULD continue sending
 the body of the request and close the stream normally.
 
 
-### Header Formatting and Compression {#header-formatting}
+### Field Formatting and Compression {#header-formatting}
 
-HTTP message headers carry information as a series of key-value pairs, called
-header fields. For a listing of registered HTTP header fields, see the "Message
-Header Field" registry maintained at
-<https://www.iana.org/assignments/message-headers>.
+HTTP messages carry metadata as a series of key-value pairs, called HTTP fields.
+For a listing of registered HTTP fields, see the "Hypertext Transfer Protocol
+(HTTP) Field Name Registry" maintained at
+<https://www.iana.org/assignments/http-fields/>.
 
-Just as in previous versions of HTTP, header field names are strings of ASCII
+Just as in previous versions of HTTP, field names are strings of ASCII
 characters that are compared in a case-insensitive fashion.  Properties of HTTP
-header field names and values are discussed in more detail in Section 3.2 of
-{{!RFC7230}}, though the wire rendering in HTTP/3 differs.  As in HTTP/2, header
-field names MUST be converted to lowercase prior to their encoding.  A request
-or response containing uppercase header field names MUST be treated as
-malformed ({{malformed}}).
+field names and values are discussed in more detail in Section 4.3 of
+{{!SEMANTICS}}.  As in HTTP/2, field names MUST be converted to lowercase prior
+to their encoding.  A request or response containing uppercase field names MUST
+be treated as malformed ({{malformed}}).
 
-Like HTTP/2, HTTP/3 does not use the Connection header field to indicate
-connection-specific header fields; in this protocol, connection-specific
-metadata is conveyed by other means.  An endpoint MUST NOT generate an HTTP/3
-message containing connection-specific header fields; any message containing
-connection-specific header fields MUST be treated as malformed ({{malformed}}).
+Like HTTP/2, HTTP/3 does not use the Connection field to indicate
```

Comment:
Again, it's perfectly OK to call it a "header field" if it hasn't said it can be used in trailers.